### PR TITLE
Write docs to go on the docs page

### PR DESCRIPTION
**Pull request by TheLunaticScripter**
Signed-off-by: John Snow <thelunaticscripter@outlook.com>

This is the start of the docs.

### CODE REVIEW

File: components/effortless-chef-io/effortless-config.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+### Policyfile Cookbook pattern
+
+This is a pattern of building an artifact for a single cookbook. 
+
+1. To use this pattern navigate to the cookbook you want to use
+   ```
+   cd chef-repo/cookbooks/foo_cookbook
+   ```
+1. Make a habitat directory
```
Comment (davymcaleer) [inline components/effortless-chef-io/effortless-config.md]: Ditto

File: components/effortless-chef-io/effortless-audit.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+> Note: To learn more about InSpec profiles checkout the documentation [here](https://www.inspec.io/docs/reference/profiles/).
+
+## Patterns
+
+### Wrapper Profile Pattern
+
+In InSpec a common pattern is to write a wrapper profile that depends on another profile. This pattern is used to pull profiles from a main profile source like the [Chef Automate Profile Store](https://automate.chef.io/docs/profiles/).
+
+1. To use this pattern navigate to the profile you want to use
```
[the remark this answers is not in the document]
Comment (TheLunaticScripter) [reply components/effortless-chef-io/effortless-audit.md]: Added this as a note.

File: components/effortless-chef-io/effortless-config.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
+# Effortless Config
+
+Effortless Config is the pattern for managing your Chef Infra workloads. It uses [Chef Habitat](https://www.habitat.sh/docs/) and [Chef Policyfiles](https://docs.chef.io/policyfile/) to build an artifact that contains the cookbooks and their dependencies alongside the scripts necessary to run them on your systems.
+
+## Patterns
+
+### Chef Repo Cookbook pattern
+
+This pattern is used to build policyfiles using the chef_repo pattern for organizing your cookbooks. The chef-repo pattern can be found [here](https://docs.chef.io/chef_repo/).
```
[the remark this answers is not in the document]
Comment (TheLunaticScripter) [reply components/effortless-chef-io/effortless-config.md]: Added as a note below this line.

File: components/effortless-chef-io/effortless-config.md (hunk 1 above)
Comment (TheLunaticScripter) [reply components/effortless-chef-io/effortless-config.md]: Same.

File: components/effortless-chef-io/effortless-audit.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+   ```
+   mkdir habitat
+   ```
+1. Make a plan file
+   
+   > Notes: For a profile targeting windows use a `plan.ps1` for Linux use a `plan.sh` if your profile targets both windows and linux you can have both a `plan.ps1` and a `plan.sh` in your habitat directory.
+   ```
+   touch plan.sh
+   ```
+1. Add some information about your profile to your plan
```
Comment (ericcalabretta) [inline components/effortless-chef-io/effortless-audit.md:29]: Should we have a powershell example here too?

File: components/effortless-chef-io/effortless-audit.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+   pkg_name=<YOUR PROFILE NAME>
+   pkg_origin=<YOUR ORIGIN>
+   pkg_version=<THE VERSION OF YOUR PROFILE>
+   pkg_maintainer="YOUR NAME AND EMAIL"
+   pkg_license=("Apache-2.0")
+   pkg_scaffolding="chef/scaffolding-chef-inspec"
+   ```
+1. Build the package
+   ```
+   hab pkg build
```
Comment (ericcalabretta) [inline components/effortless-chef-io/effortless-audit.md:40]: Where do we tell users how to load the package on a system & configure as needed?

File: components/effortless-chef-io/effortless-audit.md (hunk 4 above)
Comment (TheLunaticScripter) [reply components/effortless-chef-io/effortless-audit.md:29]: Yep that makes sense I'll add that

File: components/effortless-chef-io/effortless-audit.md (hunk 5 above)
Comment (TheLunaticScripter) [reply components/effortless-chef-io/effortless-audit.md:40]: I have that at the end after running the plan in kitchen.

File: components/effortless-chef-io/welcome.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Welcome to the Chef Effortless Patterns
+
+The Effortless Pattern is a way to better manage Chef Infra and Chef InSpec workloads using Chef Habitat and visualizing your fleet using Chef Automate. We believe that each of these technologies working together can help you better manage your infrastructure.
+
```
Comment (kagarmoe) [inline components/effortless-chef-io/welcome.md:4]: 1) Who should use the Effortless pattern? Along the lines of "If you use Chef Infra and Chef Inspec workloads to do X...
2) What are the advantages of using Effortless?? Be as specific as possible

Comment (TheLunaticScripter) [reply components/effortless-chef-io/welcome.md:4]: I added two more sections under this one that I think might answer this. Let me know what you think.

**Pull request merged**